client/src/contexts/Okteto.context.tsx

```diff
@@ -92,13 +95,25 @@ const OktetoProvider = ({ children } : OktetoProviderProps) => {
     }
   };
 
+  const getStatus = async () => {
+    if(!currentContext) return;
+    const status = await okteto.status(currentContext.name);
+    setStatus(status);
+  }
+
   useInterval(async () => {
     // Don't refresh until current command execution has finished.
     if (loading) return;
     await refreshContext();
     setReady(true);
   }, CONTEXT_POLLING_INTERVAL, true);
 
+  useInterval(async () => {
```

Comment:
This polling should only happen if an environment has been set (a docker compose file has been selected). Otherwise, we are doing unnecessary calls to the `okteto status` command every time.

Also, the `ready` state variable is meant to point out when the context is ready and this might be altering it. I would just base the execution of the "getStatus" function on whether there is an environment or not.